microcosm_pubsub/context.py

```diff
@@ -15,6 +17,10 @@ def sqs_message_context(message_dct, **kwargs):
     return context
 
 
+@defaults(
+    enable_ttl=True,
```

Comment:
You should use `<key>=typed(<type>, default_value=<value>)` here; otherwise environment variables will not be converted properly from strings.